plugins/hcloud/api_token_test.go

```diff
@@ -0,0 +1,25 @@
+package hcloud
+
+import (
+	"github.com/1Password/shell-plugins/sdk"
+	"github.com/1Password/shell-plugins/sdk/plugintest"
+	"github.com/1Password/shell-plugins/sdk/schema/fieldname"
+	"testing"
+)
+
+func TestImportingTheHetznerConfig(t *testing.T) {
+	expectedFields := map[sdk.FieldName]string{
+		fieldname.Token: "dcAuOpQaCNvjzsNPmeGXvegHBdq4Zamx8QjI8ibxfErzy34fjL4ZOITFvdP5SKct",
+	}
+
+	plugintest.TestImporter(t, APIToken().Importer, map[string]plugintest.ImportCase{
+		"Hcloud config file": {
+			Files: map[string]string{
+				"~/.config/hcloud/cli.toml": plugintest.LoadFixture(t, "hcloud.toml"),
+			},
+			ExpectedCandidates: []sdk.ImportCandidate{
+				{Fields: expectedFields, NameHint: "default"},
```

Comment:
Small thing: it would be nice for the importer to omit the name hint if it's `default`. It will get added to the 1Password item title, but in most cases, having the word "default" there doesn't provide a lot of value.

Reply:
In my example it is `default`.

In hcloud you can have multiple "context" which is similar to profile in AWS config. Each context can have an own token, so It made sense for me to show that name to the user

Reply:
Yes, so to clarify: it's perfect that you're populating the name hint with the profile name! Only suggestion is to omit the name hint if the profile name is `default`. Does that make sense?

Reply:
You can use https://github.com/1Password/shell-plugins/pull/105 for this when it's merged.

Reply:
#105 is merged.